axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -855,6 +859,22 @@
           "codeId": 843,
           "instantiateProposalId": "61"
         },
+        "sui": {
+          "maxUintBits": 64,
+          "maxDecimalsWhenTruncating": 8
+        },
+        "avalanche-fuji": {
+          "maxUintBits": 256,
+          "maxDecimalsWhenTruncating": 255
+        },
+        "optimism-sepolia": {
+          "maxUintBits": 256,
+          "maxDecimalsWhenTruncating": 255
+        },
+        "eth-sepolia": {
+          "maxUintBits": 256,
+          "maxDecimalsWhenTruncating": 255
+        },
```

Comment:
EVM ITS configs use the defaults. Not much point in listing this for 20+ evm chains, requires us to check for typos etc